t/40-script_openqa-clone-custom-git-refspec.t

```diff
@@ -103,18 +103,18 @@ $expected_re = qr/${expected}/s;
 test_once $args, $expected_re, 'clone-job command with multiple URLs in PR and job URL';
 
 $ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my/branch"}, "body": "Lorem ipsum"}'; true};
-my $needles = 'my/distri/products/sle/needles';
+my $productdir = 'products/sle';
 $ENV{curl_openqa}
-  = qq{echo -e '{"TEST": "my_test", "CASEDIR": "my/distri", "PRODUCTDIR": "distri/products/sle", "NEEDLES_DIR": "$needles"}'; true};
-$dirs = "CASEDIR=https://github.com/user/repo.git#my/branch PRODUCTDIR=repo/products/sle NEEDLES_DIR=$needles";
+  = qq{echo -e '{"TEST": "my_test", "CASEDIR": "my/distri", "PRODUCTDIR": "distri/products/sle", "NEEDLES_DIR": "$productdir"}'; true};
```

Comment:
why should the needles dir point to the product dir?

Reply:
this seems to be the case https://github.com/os-autoinst/openQA/blob/master/script/openqa-clone-custom-git-refspec#L163

Reply:
https://github.com/os-autoinst/openQA/blob/master/script/openqa-clone-custom-git-refspec#L163 sets `$old_productdir/needles`, not `$productdir` or `$old_productdir`

Reply:
is it ok if i just revert this change and close this convo?

Reply:
well, if the change is not needed, sure

Reply:
variable reverted as it was before

Reply:
Can we close this?